ucm2/sof-soundwire/sof-soundwire.conf

```diff
@@ -70,5 +70,6 @@ If.mic_init_rt715 {
 	}
 	True.BootSequence [
 		cset "name='PGA5.0 5 Master Capture Switch' 1"
+		sysw "-/class/sound/ctl-led/mic/card${CardNumber}/detach:PGA5.0 5 Master Capture Switch"
```

Comment:
this part of UCM seems to be wrong, I don't understand why we would use the SOF firmware controls for muting. This should be using the RT715 controls.

The ONLY case where SOF firmware controls should be used is for PCH-attached DMIC - because there is no codec involved. All other cases should be handled by the codecs.